### DISCUSSION TITLE
Add alphabet to Win 11 start menu?

### DISCUSSION THREAD

**Discussion by JanieKer** [Open-Shell/Open-Shell-Menu#2043] [Q&A]
My computer upgraded itself from Win 10 to Win 11 against my wishes. I installed open-shell to get the old Start menu back. I used to navigate by clicking on the letters of the alphabet in the start menu, then choosing the letter I wanted from the alphabet that popped up. The open-shell version doesn't include the letters of the alphabet. Is it possible to get them back?

**Comment by ge0rdi** [comment]
Look for `Enable accelerators` setting.

**Reply by JanieKer** [reply]
Thank you, but the Enable accelerators option (under General Behavior) was already ticked.
